yield_plugin/config/parameters.yaml

```diff
@@ -23,7 +23,7 @@ yield_max_deceleration_in_ms2: 3.0
 # Minimum safety gap with an Object/Obstacle
 # Units: meters
 # Value type: Desired
-minimum_safety_gap_in_meters: 20.0
+minimum_safety_gap_in_meters: 10.0
```

Comment:
4-10 meters is the appropriate range. Stopping 20 meters away from any object is unrealistic.